vm/src/stdlib/time.rs

```diff
@@ -704,6 +710,136 @@ mod windows {
         }
     }
 
+    fn win_perf_counter_frequency(vm: &VirtualMachine) -> PyResult<i64> {
+        let freq = unsafe {
+            let mut frequency = std::mem::MaybeUninit::uninit();
+            if QueryPerformanceFrequency(frequency.as_mut_ptr()) == 0 {
+                // TODO: modify error message more properly
+                return Err(vm.new_os_error("Failed to get performance frequency".to_owned()));
```

Comment:
Should also probably use `os::errno_err` which is the closest we currently have to `PyErr_SetFromWindowsErr` I believe.